sysl2/sysl/exporter/swagger_exporter_test.go

```diff
@@ -0,0 +1,46 @@
+package exporter
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/anz-bank/sysl/sysl2/sysl/parse"
+	"github.com/anz-bank/sysl/sysl2/sysl/syslutil"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExportAll(t *testing.T) {
+	t.Parallel()
+	modelParser := parse.NewParser()
+	const syslTestDir = "test-data"
+	files, err := ioutil.ReadDir(syslTestDir)
+	require.NoError(t, err)
+
+	for _, file := range files {
+		if file.IsDir() {
```

Comment:
This works, but this makes every file part of the one test, use `t.Run()` to run them each as subtests (see the importer tests for example), then that t.parallel() will actually be useful (as it will then run 20 of these export tests in paralllel). Not going to block the PR because of this though, please add in follow up